.github/pull_request_template.md

```diff
@@ -0,0 +1,30 @@
+# Description
+
+Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
+
+Fixes # (issue) Type the issue ID below
+
+## Type of change
+
+Please delete options that are not relevant.
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] This change requires a documentation update
+- [ ] This change requires a package install (If so, please mention the package name)
+
+# How Has This Been Tested?
+
+Have you tested your addition is not affecting any existing code and working properly?
+
+
+# Checklist:
+
+Please read and confirm all this checklist, we'll merge only if your PR get checks in all the items below. If Not applicable, please remove that particular item.
+
+- [ ] My code follows the style guidelines of this project
+- [ ] I have performed a self-review of my own code
+- [ ] I have commented my code, particularly in hard-to-understand areas
```

Comment:
Do we really need comments @naresh3599  😄

Reply:
Yes, da. It'll be good to go.